src/main/kotlin/it/pagopa/ecommerce/helpdesk/services/v1/EcommerceService.kt

```diff
@@ -116,3 +258,5 @@ class EcommerceService(
         }
     }
 }
+
+data class NTuple4<T1, T2, T3, T4>(val first: T1, val second: T2, val third: T3, val fourth: T4)
```

Comment:
why defining a custom NTuple4 data class here? 
there are the reactor already implemented [reactor Tuples.of() method](https://projectreactor.io/docs/core/release/api/reactor/util/function/Tuples.html#of-T1-T2-T3-T4-) for generating tuples

Reply:
Sorry, you are right, there was no need for a custom Tuple class. We actually reworked the logic for this task so that now there is no need at all to use it anyway, see [670ac72](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/156/commits/670ac726123f1efd9d67ce01696b2e0212740f20)